### DISCUSSION TITLE
support for authors outside of a blog plugin

### DISCUSSION THREAD

**Discussion by hellt** [squidfunk/mkdocs-material#4418] [Ideas]
I own a few documentation portals based on mkdocs-materials. Most of the doc articles are contributed by me, but I try to make docs fun inside our org and have colleagues contribute doc articles.

I think a good incentive for inside/outside frequent collaborators to see their face associated with a doc article that they contributed/authored.

This makes me think that supporting authors for a regular doc page makes sense, as this can be seen as the peronalisation aspect and incentive for collaborates to work on the docs.
I love how the author info is displayed on the blog pages, and it might be feasible to support the same for regular doc pages? What do you think?

**Comment by squidfunk** [comment]
Thanks for suggesting. This is already supported via automatic extraction from git and called [document contributors](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#document-contributors) and [document authors](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#document-authors). In theory, we could add the option to override that setting, but honestly, we already have support for two ways to list authors, so adding another one would increase maintenance overhead even more. Furthermore, documentation is a living thing – authors frequently change – while the authors of blog posts are more or less static. Managing that manually is tedious and error-prone, so our recommendation is to let one of the plugins do the heavy lifting. That's at least the use case I'm aiming for to be solved by Material for MkDocs.

As always – if your use case differs, theme extension is always an option!
